scripts/lib.sh

```diff
@@ -87,13 +87,22 @@ run-dependent-services() {
 }
 
 create-db() {
+    new_devservices_container_name=sentry-postgres-1
     container_name=${POSTGRES_CONTAINER:-sentry_postgres}
     echo "--> Creating 'sentry' database"
-    docker exec "${container_name}" createdb -h 127.0.0.1 -U postgres -E utf-8 sentry || true
+    docker exec "${new_devservices_container_name}" createdb -h 127.0.0.1 -U postgres -E utf-8 sentry \
```

Comment:
I tested this locally and works pretty well, but feels sorta weird that we display this if the new postgres container has db `sentry` already created. Anyhow, don't think there is a good solution there now and don't think we should block on that

```
createdb: error: database creation failed: ERROR:  database "sentry" already exists
Error response from daemon: No such container: sentry_postgres
```